src/mapscript/python/pymodule.i

```diff
@@ -82,13 +82,17 @@ CreateTupleFromDoubleArray( double *first, unsigned int size ) {
   $2 = &nListSize;
 }
 
-%typemap(argout,fragment="t_output_helper,CreateTupleFromDoubleArray") (double** argout, int* pnListSize) 
+%typemap(argout,fragment="CreateTupleFromDoubleArray") (double** argout, int* pnListSize) 
 {
    /* %typemap(argout) (double* argout, int* pnListSize)  */
   PyObject *r;
   r = CreateTupleFromDoubleArray(*$1, *$2);
   free(*$1);
-  $result = t_output_helper($result,r);
+  %#if SWIG_VERSION >= 0x040300
+    $result = SWIG_Python_AppendOutput($result,r,1);
```

Comment:
Did you try the following ? That's what they suggest in SWIG readme:
```suggestion
    $result = SWIG_Python_AppendOutput($result,r,$isvoid);
```

Reply:
I tried that originally, but without the SWIG version check. Updated now.